babel.config.js

```diff
@@ -0,0 +1,42 @@
+const getPlugins = ({
+	isTypescript = false,
+	isTSX = false,
+} = {}) => [
+	'./config/babel-plugin-xiva-dependency',
+	isTypescript && ['@babel/plugin-transform-typescript', {
+		isTSX,
+		allowDeclareFields: true,
+	}],
+	['@babel/plugin-proposal-decorators', {legacy: true}],
+	['@babel/plugin-proposal-class-properties', {loose: true}],
+	'babel-plugin-macros',
+	'babel-plugin-lodash',
+	'./locale/babel-plugin-transform-react.js',
+	['@babel/plugin-transform-runtime', {
+		corejs: {version: 3},
+		// TODO: Enable for web build only?
+		// useESModules: true,
```

Comment:
just enable this, preset-env will handle the "real" module stuff

Reply:
Ah, on fiddling I remember - I'm running the webpack config (`.ts`, so needs transpilation) through babel, which is using this config as well. Enabling that (even with `preset-env`) throws errors, as webpack/node doesn't handle the output. Thanks for the heads up to config in `preset-env` though, I've thrown together a caller condition for `@babel/register` that catches the WP config transpiles and swaps target to `node`.